test/modules/aws-cloudwatch-integration.ts

```diff
@@ -18,11 +18,15 @@ afterAll(async () => await execComposeDown(modules));
 describe('AwsCloudwatch Integration Testing', () => {
   it('creates a new test db', (done) => void iasql.connect(
     dbAlias,
-    process.env.AWS_REGION ?? 'barf',
-    process.env.AWS_ACCESS_KEY_ID ?? 'barf',
-    process.env.AWS_SECRET_ACCESS_KEY ?? 'barf',
     'not-needed', 'not-needed').then(...finish(done)));
 
+  it('installs the aws_account module', install(['aws_account']));
+
+  it('inserts aws credentials', query(`
+    INSERT INTO aws_account (region, access_key_id, secret_access_key)
+    VALUES ('${process.env.AWS_REGION}', '${process.env.AWS_ACCESS_KEY_ID}', '${process.env.AWS_SECRET_ACCESS_KEY}')
```

Comment:
minor, but we need to change the `CONTRIBUTING.md` to reflect this change?

Reply:
Yep! It's "simpler" as an explanation, now, too. (Connect has nothing special about it, then you connect to the database and install `aws_account`, insert credentials, and then any other modules you want, but no weird curl syntax anymore!)